### DISCUSSION TITLE
📝Comments of [blog/remark-admonitions]

### DISCUSSION THREAD

**Discussion by giscus** [PresentKim/legacy-blog-sveltekit#9] [Announcements]
# 📝Comments of [blog/remark-admonitions]

블로그에 remark-admonitions를 추가했습니다. 이제 블로그 포스트에서 아래와 같은 문법으로 정보 블럭을 사용할 수 있습니다.

https://present.kim//posts/blog/remark-admonitions

**Comment by PresentKim** [comment]
Comment!!

**Comment by PresentKim** [comment]
![](https://camo.githubusercontent.com/f9a322c724f1cbb47a2bbb5407a1abbd9b1f2a7481f0fce08bd177b59719e1b9/68747470733a2f2f6f63746f6465782e6769746875622e636f6d2f696d616765732f68756c615f6c6f6f705f6f63746f64657830332e676966)

**Comment by PresentKim** [comment]
```typescript
import css from '$lib/assets/styles/giscus/index.scss';

export const GET = async () => {
  const headers = new Headers();
  headers.append('Content-Type', 'text/css');
  return new Response(css, { headers: headers });
};
```
